src/indev/lv_indev.c

```diff
@@ -1247,6 +1259,25 @@ static void indev_proc_press(lv_indev_t * indev)
  */
 static void indev_proc_release(lv_indev_t * indev)
 {
+    if(indev->wait_until_release || /*Hover the new widget even if the coordinates didn't changed*/
+       (indev->pointer.last_point.x != indev->pointer.act_point.x ||
+        indev->pointer.last_point.y != indev->pointer.act_point.y)) {
+        lv_obj_t ** last = &indev->pointer.last_hovered;
```

Comment:
Let's use `lv_obj_t * last = indev->pointer.last_hovered;`, and change below one to `indev->pointer.last_hovered = hovered;`

Reply:
Well, set NULL in lv_obj_delete as @kisvegabor mentioned is prefer to checking valid which is more like a hacking one. What do you think?